assets/js/components/wp-dashboard/WPDashboardPopularPages.js

```diff
@@ -77,49 +78,39 @@ const WPDashboardPopularPages = ( { WidgetReportZero, WidgetReportError } ) => {
 		return <WidgetReportZero moduleSlug="analytics" />;
 	}
 
-	const headers = [
-		{
-			title: __( 'Title', 'google-site-kit' ),
-			tooltip: __( 'Page Title', 'google-site-kit' ),
-			primary: true,
-		},
-		{
-			title: __( 'Pageviews', 'google-site-kit' ),
-			tooltip: __( 'Pageviews', 'google-site-kit' ),
-		},
-	];
-	const links = [];
-	const dataMapped = data[ 0 ].data.rows.map( ( row, i ) => {
-		const [ title, url ] = row.dimensions;
-		links[ i ] = url;
-
-		return [
-			title,
-			numFmt( row.metrics[ 0 ].values[ 0 ], { style: 'decimal' } ),
-		];
-	} );
-
-	const options = {
-		chartsEnabled: true,
-		links,
-		cap: 5,
-		showURLs: true,
-		useAdminURLs: true,
-	};
-
-	const dataTable = getDataTableFromData( dataMapped, headers, options );
-
 	return (
 		<div className="googlesitekit-search-console-widget">
 			<h2 className="googlesitekit-search-console-widget__title">
 				{ __( 'Top content over the last 28 days', 'google-site-kit' ) }
 			</h2>
 			<TableOverflowContainer>
-				{ dataTable }
+				<ReportTable
+					rows={ data[ 0 ].data.rows }
+					columns={ tableColumns }
+					limit={ 5 }
+				/>
 			</TableOverflowContainer>
 		</div>
 	);
 };
 
+const tableColumns = [
+	{
+		title: __( 'Title', 'google-site-kit' ),
+		description: __( 'Page Title', 'google-site-kit' ),
+		primary: true,
+		Component: ( { row } ) => {
+			const [ title, path ] = row.dimensions;
+			return <DetailsPermaLinks title={ title } path={ path } />;
+		},
+	},
+	{
+		title: __( 'Pageviews', 'google-site-kit' ),
+		description: __( 'Pageviews', 'google-site-kit' ),
+		field: 'metrics.0.values.0',
+		Component: ( { fieldValue } ) => <Decimal value={ fieldValue } />,
```

Comment:
See above. We could simply use `Fragment` and `numFmt` here, that would simplify the implementation by reducing that extra abstraction.

Reply:
`Fragment` isn't necessary here since we're returning a single root value/node.